packages/jest-runner/src/index.ts

```diff
@@ -208,6 +287,8 @@ class TestRunner {
     };
     return Promise.race([runAllTests, onInterrupt]).then(cleanup, cleanup);
   }
+
+  on = this.UNSTABLE_eventEmitter.on.bind(this.UNSTABLE_eventEmitter);
```

Comment:
do we need the `bind`? Does it work without it?

Reply:
It was asking for an additional parameter `listener` without `bind`

Reply:
👍